library/src/scripts/toolTip/ToolTip.tsx

```diff
@@ -25,7 +25,7 @@ const nubPosition = (triggerRect, hasOverflow) => {
     };
 };
 
-function TriangleTooltip(props: { children: React.ReactNode; label: string; ariaLabel?: string }) {
+function TriangleTooltip(props: { children: React.ReactNode; label: React.ReactNode; ariaLabel?: React.ReactNode }) {
```

Comment:
```suggestion
function TriangleTooltip(props: { children: React.ReactNode; label: React.ReactNode; ariaLabel?: string }) {
```

This should be a string.